table_processing/GeneratedTable.py

```diff
@@ -5,41 +5,47 @@
 import pandas as pd
 import numpy as np
 import random
+import logging
 
 myDB= dbgen.pydb()
-
-geometry_options = {'margin': '0.7in'}
 col_list = ['country', 'city', 'zipcode', 'latitude', 'longitude','name_month', 'weekday', 'year', 'time', 'date', 'email','company', 'job_title', 'phone', 'license_plate'
+           ]
             
-]
-
 class GeneratedTable:
-    def __init__(self, rows=10, columns=5, row_lines=None, vertical_lines=None, margin='0.7in', multi_row=False, row_height=None): 
+    def __init__(self, rows=10, columns=5, row_lines=None, vertical_lines=None, margin='0.7in', multi_row=False, row_height=1.25, font_size=12, landscape=False): 
         self.rows = rows
         self.columns = columns 
         self.row_lines = row_lines
         self.vertical_lines = vertical_lines
+        self.font_size = font_size
         self.geometry_options = {
             'margin':margin,
-            'landscape':True
+            'landscape':landscape
         }
         self.multi_row = multi_row
-        self.row_height = row_height
+        if row_height < 1.25:
+            self.row_height = 1.25
```

Comment:
I'm a bit wary of magic numbers. Can these be changed to meaningfully named constants?